Assets/Scripts/Models/Furniture.cs

```diff
@@ -183,7 +184,7 @@ protected Furniture(Furniture other)
     {
         this.objectType = other.objectType;
         this.Name = other.Name;
-        this.baseType = other.baseType;
+        this.typeTags = new HashSet<string>(other.typeTags);
```

Comment:
I was wondering about the Furniture copy constructor, but @koosemose explained it to me.